tests/Extensions/Hosting.Services.Web.UnitTests/ActivityEnrichmentMiddlewareTests.cs

```diff
@@ -39,11 +39,11 @@ Task RequestDelegate(HttpContext context)
 			}
 
 			HttpContext context = new DefaultHttpContext();
-			ActivityEnrichmentMiddleware middleware = new ActivityEnrichmentMiddleware(RequestDelegate);
+			IMiddleware middleware = new ActivityEnrichmentMiddleware();
 
 			Activity activity = new Activity("TestName").Start();
 
-			await middleware.InvokeAsync(context);
```

Comment:
Does it make sense to have ConfigureAwait(false) here?

Reply:
I don't think it will change anything but I'll added it everywhere, thank you